### ISSUE TITLE
compile_sync_naive not working properly

### ISSUE THREAD

**Issue by CCherry07** [phoenix-ru/fervid#32]
cargo.toml:
<img width="567" alt="image" src="https://github.com/phoenix-ru/fervid/assets/79909910/68dbdf02-660c-422e-947f-02c52906727c">

code:
```rs
use fervid::compile_sync_naive;

use super::CompilerParams;

pub fn vue_compiler(param: CompilerParams) -> String {
  let CompilerParams { svg, .. } = param;
  compile_sync_naive(&svg, true).unwrap()
}

#[cfg(test)]
mod tests {
  use super::*;

  #[test]
  fn test_vue_compiler() {
    let svg = r#"<template>
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
    <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm-2-6h4v-2h-4v2z"/>
  </svg>
    </template>"#;
    let result = vue_compiler(CompilerParams {
      svg: svg.to_string(),
      root_path: "".to_string(),
    });
  }
}

```

err:
<img width="1103" alt="image" src="https://github.com/phoenix-ru/fervid/assets/79909910/26e16132-46da-48aa-b0d6-f574663fcb57">

**Comment by phoenix-ru** [comment]
Hi, thank you for your report. I haven't published the crate for quite a long time and API may be outdated. I can publish a new version, as it is definitely more mature :)

**Comment by phoenix-ru** [comment]
@CCherry07 I have published a new version `0.2.0`, could you please check it out? :slightly_smiling_face:

**Comment by CCherry07** [comment]
> @CCherry07 I have published a new version `0.2.0`, could you please check it out? 🙂
> 
> I would also recommend you use `fervid::compile` when you need more control, as `compile_sync_naive` is mostly used for WASM use-cases

thank you !!!

**Comment by CCherry07** [comment]
> @CCherry07 I have published a new version `0.2.0`, could you please check it out? 🙂
> 
> I would also recommend you use `fervid::compile` when you need more control, as `compile_sync_naive` is mostly used for WASM use-cases

The 0.2.0 version runs very well, thank you for your help.
